modules/_labs/core/react/index.ts

```diff
@@ -5,4 +5,5 @@ import CanvasProvider from './lib/CanvasProvider';
 export default type;
 export {type, space, CanvasProvider};
 export * from './lib/type';
+export * from './lib/StaticStates';
```

Comment:
do we still want to export this?

Reply:
I think it just makes our imports cleaner. Plus it's labs, we can always remove it if we want.

Reply:
ok sounds good to me!